packages/extension-sdk-react/src/components/ExtensionConnector/ExtensionConnector.spec.tsx

```diff
@@ -2,7 +2,7 @@
 
  MIT License
 
- Copyright (c) 2021 Looker Data Sciences, Inc.
+ Copyright (c) 2022 Looker Data Sciences, Inc.
```

Comment:
Looked at package.json and it was updated to reflect 2023. I think it may also be smarter and not update unnecessarily. This file was updated (created?) last year so year would be correct if we were still updating unnecessarily. Will leave.